clusterman/draining/queue.py

```diff
@@ -438,6 +438,7 @@ def process_queues(cluster_name: str) -> None:
 
     logger.info("Polling SQS for messages every 5s")
     while True:
+        kube_operator_client.reload_client()
```

Comment:
This may be set to `None`. Also I feel like that since this loop is much fast than the autoscaler, refreshing the client ever 5s (+ queue processing time) is a bit much.

Reply:
Yeah, it will run every 5s, but I checked performance of `reload_client`. it just loads configuration file (~2 ms)